### ISSUE TITLE
NMMNH Paleo - sub localities?

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/data-migration#213]
@Nicole-Ridgwell-NMMNHS locality L-3845 has a bunch of records with remarks like

3N 11E
4S3E
6S2E central trench
7S2E 12A5

It seems like these are grid sections of the larger site? If so, I suggest we request a new [event attribute type](https://arctos.database.museum/info/ctDocumentation.cfm?table=CTCOLL_EVENT_ATTR_TYPE):

grid section identifier - An identifier that indicates a grid section of a paleontological or archaeological dig site.

**Comment by Jegelewicz** [comment]
For now, these are in collection object remarks.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Yes, I think event attribute would be a fine place to try to move this information.

**Comment by Jegelewicz** [comment]
I have added these as Locality ID other identifiers for the moment, although some may have been left behind in object remarks and some may be original identifiers as it is difficult for me to know sometimes if they refer to a location or the object.
